### DISCUSSION TITLE
Question: Save for offline?

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [Difegue/Stylophone#70] [Q&A] [answered]
I am currently looking around at the various music streaming servers out there. For video, I will use Jellyfin - but music integration to it feels a little lackluster and more like, "its there". So eventually I came here and I really like the UI of the client, looks absolutely beautiful!

However, amongst the README and description, I couldn't see if I would be able to save certain stuff  - like a playlist - to my local disk to listen to them offline? I travel a lot with my laptop and would love to pull down songs I want to listen to, knowing that the train wifi is probably not going to perform too well :)

Thanks and kind regards,
Ingwie

**Comment by Difegue** [comment, marked as answer]
👋 Stylophone doesn't really have any music-playing capabilities on its own (local playback really just plays back an .ogg stream from the server), I don't think there's any MPD client out there with an actual offline mode of sorts?   

This would probably be better served by a more classic music player, sorry!
